### PR TITLE
updating front page for docs

### PR DESCRIPTION
**Pull request by akristen**
- removed yarn 3.2.0 pkg manager from file
- Added css magic to some file at some point
- Changed out I/O video for generic observability video
- Updated content from I/O to links to the docs site

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -66,53 +61,43 @@ const IndexPage = ({ data, pageContext, location }) => {
...
-                <br />
-                <br />
-                Didn't find what you're looking for? You're always{' '}
-                <a href="https://github.com/newrelic/newrelic-quickstarts">
-                  welcome to add contributions
-                </a>{' '}
-                or improve existing quickstarts to help others instantly monitor
-                their services.
+                New Relic is an observability platform that helps you accurately
+                capture data critical to the success of your team. Whether
```
Comment (bradleycamacho) [inline src/pages/index.js:65]:
```suggestion
                New Relic is an observability platform that helps you
                capture data critical to the success of your team. Whether
```

Comment (bradleycamacho) [reply src/pages/index.js:65]: Not sure this first sentence is the best sell of the platform.

File: src/pages/index.js [hunk 2]
```diff
@@ -66,53 +61,43 @@ const IndexPage = ({ data, pageContext, location }) => {
...
+                you're a developer installing your first agent or you're ready
+                to query ingested data, our docs site can help you find the best
+                solution for your performance needs:
```
Comment (bradleycamacho) [inline src/pages/index.js:68]: Appealing to different audience is great here, but I think there's a bit of whiplash going on.

The user lands on the dev site, but then instantly gets talked to about the docs site. That's the goal, but I think this intro should try to orient them a bit about why we're linking them over to the docs site AND specifically mention the docs site is a different site.

**Review by bradleycamacho — CHANGES_REQUESTED** [review]

**Comment by bradleycamacho** [comment]
Screen Shot 2023-01-30 at 3 35 16 PM

I'm also a bit iffy on the general look of the homepage with this content. I wonder if we should add an image/video to the right side of the content. Maybe a big signup button? Just seems a bit empty and not whimsical.

File: src/pages/index.js (hunk 1 above)
Comment (akristen) [reply src/pages/index.js:65]: Yeah, it's a bit of a hodge podge from existing content (blog post about DevOps/New Relic + Welcome to NR doc). Wanted it to be specific to what a DevOps team might need and guide them to the docs site for their answers

File: src/pages/index.js (hunk 2 above)
Comment (akristen) [reply src/pages/index.js:68]: maybe changing 'our' in the links to 'Quickstart' blahblahblah to 'the Quickstart blahblahblah on the New Relic docs site' instead. Or some kind of 'docs site' call out specifically.

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**